File: week-02/day-01/draw_diagonal.py
# Write a program that reads a number from the standard input, then draws a
# square like this:
#
#
# %%%%%
# %%  %
# % % %
# %  %%
# %   %
# %%%%%
#
# The square should have as many lines as the number was
lines = int(input("Number of lines: "))
puffer = ""

for k in range(0, lines):
   for i in range(0, lines):
       if k == 0 or k == lines - 1:
           puffer += "%"
       else:
           puffer = "%"
           for l in range(1, lines - 1):
               if l == k:
                   puffer += "%"
               else:
                   puffer += " "
           puffer += "%"
   print (puffer + "\n")
   puffer = ""

File: week-08/day-01/32_draw_diagonal/draw_diagonal.js
// Write a program that draws a
// square like this:
//
//
// %%%%%
// %%  %
// % % %
// %  %%
// %   %
// %%%%%
//
// The square should have as many lines as lineCount is

var lineCount = 6;
var line ="";
for(i=0; i<=lineCount; i++){
    for(j=0; j<=lineCount; j++){
        if(j===0 || i===0 || j===lineCount || i===lineCount || i===j){
            line += "%";
        }else{
            line += " ";
        }
    }
    console.log(line);
    line="";
}